data/src/main/java/org/oppia/data/backends/gae/model/GaeTopicSummarytDict.kt

```diff
@@ -0,0 +1,24 @@
+package org.oppia.data.backends.gae.model
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+
+/**
+ * Data class for TopicSummaryDict model containing full information
+ * https://github.com/oppia/oppia/blob/77d6fd/core/domain/topic_domain.py#L1178
+ */
+@JsonClass(generateAdapter = true)
+data class GaeTopicSummarytDict(
```

Comment:
This isn't actually a dictionary, it's an object. Is 'GaeTopicSummary' suitable here?

NB: This is a bit nitpicky, but 'dict' is a python structure and should stay probably isolated to python contexts since it's a bit confusing form the Java perspective.

Reply:
Okay got it. Will change accordingly and also inform will @veena14cs about this.